packages/common/src/search/types/IHubSearchResult.ts

```diff
@@ -0,0 +1,110 @@
+import { IHubGeography } from "../..";
+
+/**
+ * Standardized light-weight search result structure, applicable to all
+ * types of search results - users, groups, content, events etc
+ */
+export interface IHubSearchResult {
```

Comment:
Seems like this could have inherited from `IHubEntityBase`. Is there a reason you didn't do that? If it's just for the comments, I'd say you could update the comments on that interface.

Reply:
If there is clean overlap I'll extend the base... definitely better than re-defining.

Reply:
Extended from IHubEntityBase and updated comments in that.